### PR TITLE
Add curator component

### PR DESCRIPTION
**Pull request by 3Mydlo3**
**When merged this pull request will:**
- Add curator component
- Add automatic curator assignment for designated units

### CODE REVIEW

**Comment by veteran29** [comment]
I suppose that editor attribute will be added separately?

File: addons/curator/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Curator
+
+Our adjustments and upgrades for Zeus.
+
+### Automatic Zeus assignment
+
+Some missions involve more than one Zeus. It is hard to get it working correctly and without problems when one Zeus disconnects. Not anymore. All units with variable `afm_curator_isZeus` set to true on init will have curator module assigned on startup (and JIP).
+
+## Authors
```
Comment (veteran29) [inline addons/curator/README.md]: I've noticed that I'm using `###` for that. 🤔

**Comment by 3Mydlo3** [comment]
I thought I will add editor attribute later, but now I see it does not require too much work, so I will add it in this PR.

File: addons/curator/README.md (hunk 1 above)
Comment (3Mydlo3) [reply addons/curator/README.md]: `back_to_game` and `friendly_tracker` also use `##`, we could fix this in separate PR along with `requiredAddons` entry.

Comment (veteran29) [reply addons/curator/README.md]: Yeah, I agree. We'll do cleanup PR later. I also want to add `authors[] = {"<nicknames>"};` to CfgPatches later on.

File: addons/curator/functions/fnc_assignZeus.sqf [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * Example:
+ * [player] call afm_curator_fnc_assignZeus
+ *
+ * Public: No
+ */
+
+params ["_unit", ["_isZeus", true]];
+
+systemChat str _unit;
```
Comment (veteran29) [inline addons/curator/functions/fnc_assignZeus.sqf]:
```suggestion
```

File: addons/curator/functions/fnc_assignZeus.sqf [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+if (_isZeus) then {
+    [QGVAR(zeusConnected), [_unit]] call CBA_fnc_serverEvent;
```
Comment (veteran29) [inline addons/curator/functions/fnc_assignZeus.sqf]: IMHO these events have changed their initial meaning, maybe rename to `QGVAR(assign)`, `QGVAR(unassign)` or `QGVAR(zeusAssign)`, `QGVAR(zeusUnassign)`

**Pull request merged**